src/api/Themer.ts

```diff
@@ -0,0 +1,121 @@
+import { Theme } from "../entities";
+import { Constants, Dialog, ReactNative } from "../metro";
+
+export const themes = {} as Record<string, Theme>;
+export let currentTheme: Theme;
+export let themeApplied: boolean, themeErrorReason;
+let discordConstants: typeof Constants;
+enum ThemeType {
+    DARK,
+    LIGHT,
+    AMOLED
+}
+
+const { externalStorageDirectory } = window.nativeModuleProxy.AliucordNative;
+const SETTINGS_DIRECTORY = externalStorageDirectory + "/AliucordRN/settings/";
+const THEME_DIRECTORY = externalStorageDirectory + "/AliucordRN/themes/";
+const settings = JSON.parse(AliuFS.readFile(SETTINGS_DIRECTORY + "Aliucord.json", "text") as string);
```

Comment:
shouldn't this fail terribly on first load

Reply:
nop it works fine

Reply:
well actually yes wait